app/main.py

```diff
@@ -1 +1,63 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.skills = []
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.name = name
+        self.skills = ["JavaScript",
+                       "HTML",
+                       "CSS"
+                       ]
```

Comment:
Use skills.extend(...)
What if some skills will be adeded by default to SoftwareEngineer?